domains/narmis.is-cool.dev.json

```diff
@@ -2,7 +2,7 @@
     "description": "My portfolio hosted on GitHub Pages",
 
     "domain": "is-cool.dev",
-    "subdomain": "narmis",
+    "subdomain": "_github-pages-challenge-Narmis-E.narmis.is-cool.dev",
```

Comment:
File name needs to be changed in order for this to work as well.

.is-cool.dev is not needed.